packages/flutter/lib/src/material/text_field.dart

```diff
@@ -69,6 +69,13 @@ class _TextFieldSelectionGestureDetectorBuilder extends TextSelectionGestureDete
   void onSingleTapUp(TapDragUpDetails details) {
     super.onSingleTapUp(details);
     _state._requestKeyboard();
+  }
```

Comment:
Seems `_state.widget.onTap?.call();` is removed from `onSingleTapUp`. Just wonder if this is intended?

Reply:
This is intended, it has been moved to `onUserTap`. This is a non-breaking change because `onUserTap` will still be called with `onSingleTapUp` in `TextSelectionGestureDetector` as it was before this change. Extenders of `TextSelectionGestureDetectorBuilder` will be okay because `TextSelectionGestureDetector` retains its previous behavior as the default.